OutOfSchool/OutOfSchool.Encryption/Services/DevEUSignOAuth2Service.cs

```diff
@@ -0,0 +1,19 @@
+using System.Text.Json;
+using OutOfSchool.Encryption.Models;
+
+namespace OutOfSchool.Encryption.Services;
+
+public class DevEUSignOAuth2Service : IEUSignOAuth2Service
+{
+    /// <inheritdoc/>
+    public CertificateResponse GetEnvelopCertificateBase64() => new()
+    {
+        CertBase64 = Convert.ToBase64String("very_secret_certificate"u8.ToArray()),
+    };
+
+    /// <inheritdoc/>
+    public UserInfoResponse DecryptUserInfo(EnvelopedUserInfoResponse encryptedUserInfo) =>
```

Comment:
Since this is the public method could you please:
1. add the `[NotNull]` attribute to the argument
2. validate it is not null.